<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_bin">
      <property name="text">
       <string>Bin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_bin">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label__hex">
      <property name="text">
       <string>Hex</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_hex">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_dec">
      <property name="text">
       <string>Dec</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_dec">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_ascii">
      <property name="text">
       <string>ASCII</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ascii">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_float">
      <property name="text">
       <string>float representation</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_float">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QRadioButton" name="signed_2">
      <property name="text">
       <string>signed</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QRadioButton" name="unsigned_2">
      <property name="text">
       <string>unsigned</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
